sympy/physics/control/tests/test_lti.py

```diff
@@ -0,0 +1,103 @@
+from sympy import symbols, Matrix, expand, together
+from sympy.physics.control.lti import TransferFunction, is_proper
+
+a, b, g, s, p = symbols('a, b, g, s, p')
+
+
+def test_TransferFunction_create():
+    G = Matrix([1 / s, 1 / (s + 1)])
+    tf1 = TransferFunction(G)
+    assert tf1.G == G
+
+    G = Matrix([[s / (s**2 + 3), 1 / s],
+            [1 / (s + g), s / (s**2 - g*s - 3)]])
+    tf2 = TransferFunction(G)
+    assert tf2.G == G
+
+    G = Matrix([[(4*p - 10) / (2*p + 1), 3 / (p + 2)],
+            [1 / (2*p**2 + 5*p + 2), (p + 1) / (p**2 + 4*p + 4)]])
+    tf3 = TransferFunction(G)
+    assert tf3.G == G
+
+
+def test_TransferFunction_solve():
+    G1 = Matrix([[1 / s + 2 / s**2],
+            [2*s / (s**2 + 3*s + 5)]])
+    u1 = Matrix([s + 1])
+    assert expand(TransferFunction(G1).solve(u1) - G1*u1).is_zero
+
+    G2 = Matrix([s / (1 + s**2), 1 / s])
+    u2 = Matrix([1 / p])
+    assert expand(TransferFunction(G2).solve(u2) - G2*u2).is_zero
```

Comment:
I don't think using `is_zero` is robust. I think you have to do things like `simplify(expected - result) == 0` to get the best check for symbolic zeroness.